<?xml version="1.0"?>
<launch>
    <arg name="building" default="$(optenv BUILDING AMK)"/>
    <node name="osm_bridge_ros" pkg="osm_bridge_ros_wrapper" type="osm_bridge_ros.py" output="screen">
        <rosparam command="load" file="$(find osm_bridge_ros_wrapper)/ros/config/$(arg building).yaml" />
        <param name="overpass_server_ip" value="127.0.0.1" />
        <param name="overpass_server_port" value="8000" />
    </node>
</launch>
